src/test/groovy/com/github/vkorobkov/jfixtures/util/CollectionUtilTest.groovy

```diff
@@ -53,6 +53,17 @@ class CollectionUtilTest extends Specification {
         CollectionUtil.mapValues([:], { it * 2}).isEmpty()
     }
 
+    def "#mapValues preserves elements order"() {
```

Comment:
This issue was found thanks to integration tests! ❤️

Reply:
:+1: